rakelib/test.rake

```diff
@@ -16,6 +16,13 @@ namespace "test" do
     RSpec::Core::Runner.run(["--fail-fast", *Rake::FileList["spec/**/*.rb"]])
   end
 
+  task "plugins" => [ "bootstrap","plugin:install-all" ] do
+    require "logstash/environment"
+    gem_home = LogStash::Environment.logstash_gem_home
+    pattern = "#{gem_home}/logstash-*/spec/{input,filter,codec,output}s/*_spec.rb"
+    sh "./bin/logstash rspec #{pattern}"
+  end
```

Comment:
This call require us to be at the root of logstash, we should use the Logstash::LOGSTASH_HOME constant to generate the path so we can run this task anywhere.